### DISCUSSION TITLE
FY23 project summary

### DISCUSSION THREAD

**Discussion by jsv1206** [lsst-epo/citizen-science-notebooks#78] [General]
Becky and Sree worked with Eric, Clare, and Brian to create and contribute to many of the citizen science utilities that are now part of the rubin.citsci package. 

In addition to contributing to the features of the `Citizen_Science_Testing.ipynb` notebook, which is the main tutorial notebook in the tutorial repo, Becky also focused on pedagogy and clarity. 

Sree worked with Eric to create the `variable_stars.ipynb` notebook which queries the variable stars and the flux and plots the cutouts and lightcurves. We added a new functionality to send flipbooks to Zoonievrse.

Finally, Sree and Becky prepared, presented, and recorded a run through of said tutorial notebook to be made available on the Rubin website.

**Comment by jsv1206** [comment]
@ericdrosas87 Would you like to add anything to this summary?

**Reply by ericdrosas87** [reply]
Hey @jsv1206  - I think I understand the wording above, but I would suggest the following rewording and addition:

“Becky and Sree worked with Eric, Clare, and Brian to create and contribute to many of the citizen science utilities in the `utils.py` script. Eric also created the rubin.citsci package and iterated on it with input from Clare, Brian, Sree, and Becky. 

The rubin.citsci package is the Notebook Aspect’s touchpoint to the RSP Data Exporter service running in the EDC which, together with the rubin.citsci package, constitute the citizen science data pipeline to Zooniverse.”
